source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -569,14 +571,14 @@ void ContextImpl::logHandshake(SSL* ssl) const {
 #error "Delete preprocessor check below; no longer needed"
 #endif
 
-#if BORINGSSL_API_VERSION >= 18
+#if 0
```

Comment:
Could you reinstate the original upstream code, and then add an *additional* `#if 0` with a comment, just around the `SSL_was_key_usage_invalid()` call. That way, the intent is clearer, and we are modifying upstream code less by only making a small addition of the `#if 0`.